src/agent/srcview/src/pdbcache.rs

```diff
@@ -47,57 +133,74 @@ impl PdbCache {
                 }
             };
 
+            // Gather all inlinee entries
+            let mut inlinees = BTreeMap::new();
+            let mut inlinee_iter = info.inlinees()?;
+            while let Some(inlinee) = inlinee_iter.next()? {
+                let fname = match id_finder.find(inlinee.index())?.parse()? {
+                    pdb::IdData::Function(fid) => fid.name,
+                    pdb::IdData::MemberFunction(mid) => mid.name,
+                    _ => {
+                        bail!("Expected LF_MFUC_ID or LF_FUNC_ID for {}", inlinee.index());
```

Comment:
For ease of debugging if this ever fails, we should report what the actual `IdData` type is here